docs/use.md

```diff
@@ -187,6 +190,119 @@ The [render-as-you-fetch pattern](https://reactjs.org/docs/concurrent-mode-suspe
 
 Just as the SDK hooks like `useFirestore` can automatically fetch an SDK, you can call `preloadFirestore` (or `preloadAuth`, etc) to start loading an SDK without suspending.
 
+### Initialize an SDK
+
+A few Firestore SDKs need to be initialized (`firebase().performance().initialize()`, `firebase.remoteConfig().fetchAndActivate()`), or need to have settings set before any other calls are made (`firebase.firestore().enablePersistence()`). This can be done by using an extra argument in the preload method:
+
+```jsx
+preloadFirestore(firestore => {
```

Comment:
First arg is app, either change to `preloadFirestore(firebaseApp, firestore => {` or `preloadFirestore(undefined, firestore => {`

Reply:
fixed